recipes/c-client/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+import os
+import stat
+
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.43.0"
+
+
+class CclientConan(ConanFile):
+    name = "c-client"
```

Comment:
Nobody asked, but why this name?

Reply:
https://github.com/winlibs/imap/blob/4f5969e9dd677122f30416de044576644f612c71/docs/internal.txt#L184-L190